translate/storage/pypo.py

```diff
@@ -864,12 +862,9 @@ def _getoutput(self):
     def encode(self, lines):
         """encode any unicode strings in lines in self._encoding"""
         newlines = []
-        encoding = self._encoding
-        if encoding is None or encoding.lower() == "charset":
-            encoding = 'UTF-8'
         for line in lines:
             if isinstance(line, six.text_type):
-                line = line.encode(encoding)
+                line = line.encode(self.encoding)
```

Comment:
A few lines below, in `decode()`, `self._encoding` is being used but probably is fine to use `self.encoding` and simplify the if condition a bit.